Tests/database_tests.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+import os
+from dojo import Dojo
+
+
+class TestSaveState(unittest.TestCase):
+
+    """ Test cases for state to database(s) """
+
+    def test_successful_save_state_creates_database_without_db_name(self):
+
+        """ Test successful creation of default database file dojo.db """
+
+        self.dojo = Dojo()
+        self.assertTrue(os.path.isfile("dojo.db"))
+        os.remove("dojo.db")
+
+    def test_successful_save_state_creates_database_with_db_name(self):
+
+        """ Test successful creation of database file with db_name as its name """
+
+        self.dojo = Dojo()
+        self.dojo.save_state("january_data")
+        self.assertTrue(os.path.isfile("january_data.db"))
+        os.remove("january_data.db")
+
+    def test_save_state_succesfully_stores_data_in_database(self):
+
+        """ Test successful storage of data from app into database using the save_state method """
+
+        self.dojo = Dojo()
+        self.dojo.create_room("office", ["kenya"])
+        self.dojo.create_room("living", ["Kampala"])
+        self.dojo.add_person('Ian', 'pyr' ,'FELLOW', 'Y')
+        self.dojo.add_person('Joel', 'Ortiz' ,'FELLOW', 'Y')
+        self.dojo.add_person('Armin', 'Buren' ,'STAFF')
+        msg = self.dojo.save_state('test')
+        self.assertTrue(os.path.isfile("test.db"))
+        self.assertEqual(msg, "You have saved data to the database")
```

Comment:
I was advised checking if the file has been created isn't the best way to test this functionality. You don't know if it has the content you want. You could import sqlite3 (if it's an sqlite database) and assert that the columns contain the content you expect.